portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,16 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  display:flex;
+  flex-direction:column;
+  justify-content:center;
+  align-items:center;
```

Comment:
Can you add a space here (eg: align-items: center;) ditto for the rest of these too.

Reply:
thanks for the feedback, I've implemented these changes now 👍